ocs_ci/utility/rosa.py

```diff
@@ -58,17 +58,35 @@ def create_cluster(cluster_name, version, region):
     multi_az = "--multi-az " if config.ENV_DATA.get("multi_availability_zones") else ""
     cluster_type = config.ENV_DATA.get("cluster_type", "")
     provider_name = config.ENV_DATA.get("provider_name", "")
+    rosa_mode = config.ENV_DATA.get("rosa_mode", "")
```

Comment:
I'd suggest making auto the default rosa mode if it's not specified in the config

Reply:
OK